pkg/controller/nodeconfig/resource.go

```diff
@@ -226,11 +256,12 @@ func makeNodeSetupDaemonSet(nc *scyllav1alpha1.NodeConfig, operatorImage, scylla
 							Args: []string{
 								`
 # Create a temporary root that will represent the host file system and devices.
+# It shall contain identical file tree as the host, so all symlinks keep working,
+# and we'll add additional "virtual" data into /scylla-operator folder.
+# This will avoid polluting the node with the extra data and avoids the need to clean up. 
 cd "$( mktemp -d )"
```

Comment:
Trying to understand the context, so more of a question to the existing implementation, rather than this PR, but why do this instead of an emtpyDir + an emptyDir subpath for scylla-operator dir? You could then mount the kube-api-access volume directly, instead of binding it. Or am I missing something here?

Reply:
I guess an emptyDir + workingDir would do the same

Reply:
ok, I don't mind either approach